docs/user-interface/controls/blazorwebview.md

```diff
@@ -0,0 +1,123 @@
+
+    <b:BlazorWebView HostPage="wwwroot/index.html">
+        <b:BlazorWebView.RootComponents>
+            <b:RootComponent Selector="app" ComponentType="{x:Type local:Main}" />
+        </b:BlazorWebView.RootComponents>
+    </b:BlazorWebView>
+
+</ContentPage>
+```
+
+The root Blazor component for the app is in *Main.razor*. The rest of the Blazor components are in the *Pages* and *Shared* project folders, and are identical to the components used in the default Blazor template. Static web assets for the app are in the *wwwroot* folder.
+
+## Add a BlazorWebView to an existing app
+
+The process to add a `BlazorWebView` to an existing .NET MAUI app is as follows:
+
+1. Add the Razor SDK, `Microsoft.NET.Sdk.Razor` to your project by editing its first line:
+
+    ```xml
+    <Project Sdk="Microsoft.NET.Sdk.Razor">
+    ```
+
+    The Razor SDK is required to build and package projects containing Razor files for Blazor projects.
+
+1. Add the root Blazor component for the app to the project.
+1. Add your Blazor components to project folders named *Pages* and *Shared*.
+1. Add your static web assets to a project folder named *wwwroot*.
+1. Add any optional *_Imports.razor* files to your project.
+1. Add a `BlazorWebView` to a page in your .NET MAUI app, and point it to the root of the Blazor app:
+
+    ```xaml
+    <ContentPage xmlns=http://schemas.microsoft.com/dotnet/2021/maui
+                 xmlns:x=http://schemas.microsoft.com/winfx/2009/xaml
+                 xmlns:b="clr-namespace:Microsoft.AspNetCore.Components.WebView.Maui;assembly=Microsoft.AspNetCore.Components.WebView.Maui"
+                 xmlns:local="clr-namespace:MyBlazorApp"
+                 x:Class="MyBlazorApp.MainPage">
+
+        <b:BlazorWebView HostPage="wwwroot/index.html">
+            <b:BlazorWebView.RootComponents>
+                <b:RootComponent Selector="app" ComponentType="{x:Type local:Main}" />
+            </b:BlazorWebView.RootComponents>
+        </b:BlazorWebView>
+
+    </ContentPage>
+    ```
+
+1. Modify your `Startup` class to add `using` directives for the `Microsoft.AspNetCore.Components.WebView.Maui` and `Microsoft.Extensions.DependencyInjection` namespaces:
+
+    ```csharp
+    using Microsoft.AspNetCore.Components.WebView.Maui;
+    using Microsoft.Extensions.DependencyInjection;
+    ```
+
+    > [!NOTE]
+    > Blazor requires service scopes, which are only supported with the `Microsoft.Extensions.DependencyInjection` library.
+
+1. Modify the `Configure` method of your `Startup` class to register the `BlazorWebView` control for use in your app. To do this, call the `RegisterBlazorMauiWebView` method, the `UseMicrosoftExtensionsServiceProviderFactory` method, and the `ConfigureServices` method on the `IAppHostBuilder` object. Then, on the `IServiceCollection` object, call the `AddBlazorWebView` method to add component web view services to the services collection:
+
+    ```csharp
+    public class Startup : IStartup
```

Comment:
This pattern changed and `Startup` is gone and replaced with `MauiProgram`. Please update this section to match what's in a new MAUI Blazor app, which is somewhat different from previous previews.